start.py

```diff
@@ -34,16 +63,60 @@ def main(args):
     # Nodes in the Kafka cluster
     nodes = [Node(hostname=hostname,
                   group='brokers',
-                  ports=[2181, 9092],
+                  ports=[ZOOKEEPER_PORT, BROKER_PORT],
                   image=image)
              for hostname in args.brokers]
 
     cluster = Cluster(*nodes)
     cluster.start(args.network, pull_images=args.always_pull)
 
-    # TODO: Add support for cluster mode (e.g. all nodes are part of the same cluster). Today we only
-    # start each node independently so they will all end up independent one-node clusters.
+    # Create distributed zookeeper configuration
+    zookeeperConfig = 'tickTime=2000\n' \
```

Comment:
Follow Python variable name convention. Do this throughout `zookeeper_config` instead of `zookeeperConfig`.

Use parentheses instead of trailing slashes

```
zookeeper_config = ('tickTime=2000\n'
                    'blahblahblah')
```